cmd/updater/update.sh

```diff
@@ -374,6 +374,11 @@ function run_systemd_action() {
                 echo "systemd system service: $action"
                 return 0
             fi
+        else
+            if sudo -n systemctl "$action" "algorand@$(systemd-escape "$data_dir")"; then
+                echo "systemd system service: $action"
```

Comment:
Should this command be with sudo, `sudo systemd system service: $action`

Reply:
While not affecting functionality, it would be nice in the debug to indicate it was run with sudo.